crossplane-capi-import/README.md

```diff
@@ -0,0 +1,301 @@
+# Importing EKS/AKS/GKE clusters to CAPI using crossplane
+
+## Introduction
+
+Team Honeybadger has proposed a solution to importing existing clusters to CAPI
+via the use of crossplane in place of writing our own custom importer.
+
+The purpose of this is to provide a mechanism whereby customers may come to
+Giant Swarm on a Bring Your Own Cluster (BYOC) basis such that we can support
+and manage apps on the customers cluster without compromising the integrity of
+their existing infrastructure deployment mechanisms. That is to say, customers
+operate and manage their own infrastructure, whilst we provide support and
+deployment strategies for our own applications to run inside the customers
+cluster.
+
+Throughout this RFC, where the words __import__ and/or __adopt__ are used, they
+are to be read as "unmanaged".
```

Comment:
I'd rather we agree to only use "import" for unmanaged, as we tried to define "adopt" as "we adopt and manage lifecycle"

Reply:
Agreed. I added a clarification here that `import` refers to unmanaged clusters whilst adopt refers to full management.